ui/app/routes/vault/cluster/unload-model-record.js

```diff
@@ -0,0 +1,46 @@
+import Route from '@ember/routing/route';
```

Comment:
new base class to replace `UnloadModelRoute` mixin. Thank you @zofskeez for helping!